scripts/build.sh

```diff
@@ -409,7 +409,7 @@ build_deb_packages() {
 		# Check if we need to rebuild this package
 		SOURCEHASH=$(cd ${SOURCES}/${NAME} && git rev-parse --verify HEAD)
 		if [ $NAME != truenas -a -e "${HASH_DIR}/${NAME}.hash" ] ; then
-			if [ "${KMOD}" = "true" ] && [ -n "${KERN_UPDATED}" ]; then
+			if [ "${KMOD}" = "true" ] && [ "${KERN_UPDATED}" -eq "1" ]; then
```

Comment:
JFYI, -eq is for numerical comparison, should use without quotes, or use `=`

Reply:
Thank you William for pointing it out, will be fixed by https://github.com/truenas/scale-build/commit/da850801b66f889ea05d6239860e922e09043db6